.github/workflows/docs_cleanup.yml

```
@@ -0,0 +1,25 @@
+name: Cleanup Docs
+
+on: delete
+
+jobs:
+  Clean-Up-Test-Docs:
+    runs-on: ubuntu-20.04
+    # permissions:
+    #   contents: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Clean up docs
+        run: |
+          echo "Clean up for branch ${{ github.event.ref }}"
+          git fetch
+          git checkout gh-pages
+          git rm ${{ github.event.ref }}
```

```suggestion
          git rm -r ${{ github.event.ref }}
```
Isn't `-r` flag needed?

---

thanks for your review! will update it.